fix_equals.py

```
@@ -0,0 +1,26 @@
+import re
+
+def fix_file(filepath):
+    with open(filepath, 'r') as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)